package.json

```diff
@@ -139,10 +139,10 @@
     "static-site-generator-webpack-plugin": "^3.4.1",
     "style-loader": "^0.18.2",
     "tap-spot": "^1.1.1",
-    "terser-webpack-plugin": "^1.2.3",
+    "terser-webpack-plugin": "^2.3.5",
```

Comment:
It is ok to remove it, right? It is not longer used.

Reply:
@montogeek it is used, actually bundlesize growth and build failure yesterday due to removing it. Almost 2x gzipped without passing it directly as a minimizer as i wrote above, i was sure its safe to remove it and simply do `optimization.minimize: true`.

Whats interesting in lockfile for webpack 4.42 1.4 version of it used, so the one we pass, is actually next major v